team/jesslyn-difiori

```diff
@@ -0,0 +1,21 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Jesslyn Di Fiori
+  avatar: https://avatars.githubusercontent.com/jkdifiori
+  role: Technical Writer
+  organizations:
+    - name: National Snow and Ice Data Center (NSIDC)
+  bio: >
+    A FEW SENTENCES ABOUT YOU, MULTIPLE LINES ARE FINE
```

Comment:
thanks @jkdifiori don't forget to modify these other sections in CAPS before we can merge your bio